.pre-commit-config.yaml

```diff
@@ -0,0 +1,20 @@
+# HOWTO: https://pre-commit.com/#usage
+# pip3 install pre-commit
+# pre-commit install
+
+repos:
+  - repo: https://github.com/prettier/prettier
+    rev: 2.0.5
+    hooks:
+      - id: prettier
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v3.1.0
+    hooks:
+      - id: check-merge-conflict
+      - id: detect-private-key
+  - repo: https://github.com/packit/pre-commit-hooks
+    rev: master
```

Comment:
Use exact hash (like [you did in ogr](https://github.com/packit/ogr/commit/a8d4d469af85057433d2cc03d6fadd4a663146b2)) instead of this moving `master`